src/pkg/utils/utils.go

```diff
@@ -707,24 +711,29 @@ func ResolveImageName(distro, image, release string) (string, string, error) {
 	logrus.Debugf("Image: '%s'", image)
 	logrus.Debugf("Release: '%s'", release)
 
+	if viper.IsSet("general.image") {
+		image = viper.GetString("general.image")
```

Comment:
This doesn't look right, because it will override the `--image` command line option with the `image` option from the configuration file.

Explicitly specified command line options should always have priority over the configuration file.

Reply:
You're right.